mappings/net/minecraft/class_7428.mapping

```diff
@@ -0,0 +1,25 @@
+CLASS net/minecraft/class_7428
+	FIELD comp_744 expiresAt Ljava/time/Instant;
+	FIELD comp_745 keyString Ljava/lang/String;
+	FIELD comp_746 signature Ljava/lang/String;
+	FIELD field_39051 PUBLIC_KEY Ljava/lang/String;
+	METHOD <init> (Lcom/mojang/datafixers/util/Pair;Ljava/lang/String;)V
+		ARG 1 expiresAtKeyString
+		ARG 2 signature
+	METHOD comp_744 expiresAt ()Ljava/time/Instant;
+	METHOD comp_745 keyString ()Ljava/lang/String;
+	METHOD comp_746 signature ()Ljava/lang/String;
+	METHOD method_43548 getAsProperty ()Lcom/mojang/authlib/properties/Property;
+	METHOD method_43549 (Lcom/mojang/authlib/GameProfile;)Ljava/util/Optional;
+		ARG 0 gameProfile
+	METHOD method_43550 (Lcom/mojang/authlib/minecraft/MinecraftSessionService;)Lnet/minecraft/class_7428$class_7429;
+		ARG 1 sessionService
+	METHOD method_43552 (Ljava/lang/String;)Ljava/util/Optional;
+		ARG 0 keyString
+	METHOD method_43554 isNotExpired ()Z
+	METHOD method_43555 (Lcom/mojang/authlib/GameProfile;)Lcom/mojang/authlib/GameProfile;
+		ARG 1 gameProfile
+	CLASS class_7429
+		FIELD comp_747 publicKey Ljava/security/PublicKey;
+		METHOD comp_747 publicKey ()Ljava/security/PublicKey;
+		METHOD method_43557 createSignatureInstance ()Ljava/security/Signature;
```

Comment:
```suggestion
		METHOD method_43557 createSignature ()Ljava/security/Signature;
```

Reply:
I'd prefer Instance, because no data is passed. `createSignature` means, that you kind of sign something.